conf/config.json

```
@@ -1,5 +1,8 @@
 {
-    "default_server_name": "__DEFAULT_HOME_SERVER__",
+      "default_server_config": {
+        "m.homeserver": { "base_url": "__DEFAULT_HOME_SERVER__" },
```

```suggestion
        "m.homeserver": { "base_url": "https://__DEFAULT_HOME_SERVER__" },
```
We may need to add `https://` otherwise I get `base_url pour m.homeserver non valide`